### ISSUE TITLE
Any way to disable the add and tune buttons?

### ISSUE THREAD

**Issue by Slayking** [codex-team/editor.js#1426]
Hi, I'm wondering if there's anyway to disable the add and tune buttons.

It's not really necessary for what I am building and they are cluttering the already small text box. I really only want to use the inline tool bar. There's not too many resources for this editor so I thought I'd ask here.

Thanks :D

**Comment by ryandunnewold** [comment]
@Slayking I am disabling it with CSS. It's not ideal, but it works:

```
.ce-toolbar {
  display: none !important;
}
```

**Comment by Slayking** [comment]
Works for now, not ideal yeah, unfortunate there's no existing config, hope the devs add it in the future. I was initially trying to disable it with js by deleting the element but that didn't work.

**Comment by milu2003** [comment]
i need it too

**Comment by lofcz** [comment]
I second this request @neSpecc would you please take a look at this?

**Comment by aakay** [comment]
yes, please consider adding this feature. We sometimes need blocks which should be pinned atop and never touched.

**Comment by CodeLearningSlayer** [comment]
Is there still no solution?

**Comment by brian7989** [comment]
Any solution?

**Comment by nicoschuck** [comment]
> yes, please consider adding this feature. We sometimes need blocks which should be pinned atop and never touched.

I agree. This feature would be very helpful!

**Comment by m-burger** [comment]
Hello, still no news about this request ?

We also could use this feature as some blocks shouldn't be able to use tools in some use cases.

**Comment by e-afzal** [comment]
I second this. Is there still no way to hide these without CSS using a property where you can set a boolean value like the 'hideToolbar' property which is already available?

**Comment by neSpecc** [comment]
I think, we can add it. We have legacy `hideToolbar` option that worked for that reason in the 1.0 version. We can make it work again like this.

**Comment by e-afzal** [comment]
> I think, we can add it. We have legacy `hideToolbar` option that worked for that reason in the 1.0 version. We can make it work again like this.

Thanks for the reply @neSpecc! I have applied the CSS solution by @ryandunnewold and it worked great. But if there's the hideToolbar like feature, that would be super swell. 👍🏻
